cmd/cloudx/proxy/helpers.go

```diff
@@ -125,16 +125,11 @@ func runReverseProxy(ctx context.Context, h *client.CommandHelper, stdErr io.Wri
 		}
 	}()
 
-	// TODO: we probably don't want to support this in the future, but it requires to be authenticated
-	slug := os.Getenv("ORY_PROJECT_SLUG")
-	if slug == "" {
-		project, err := h.GetSelectedProject(ctx)
-		if err != nil {
-			return err
-		}
-		slug = project.Slug
```

Comment:
🎉

Reply:
Do we now always require an api key to be able to use the proxy and tunnel command?

Reply:
I'm asking because in many of our quickstarts we use this method for set up: https://www.ory.sh/docs/getting-started/integrate-auth/go#test-your-application

I just tried it locally on my system and it triggers an auth screen when I run `ory proxy http://localhost:3000`. Using `-q` doesn't work: `can not sign in or sign up when flag --quiet is set`.

This used to work just fine without auth (social sign in however did not work!) but now requires auth always.

I think the question is where we go from where. One the one side I like the idea of making it "just work" also for social sign in.

Reply:
The problem with anonymous use is that depending on the use-case we need either the project slug or the project ID. IMO it is confusing when you sometimes have the option to provider either, and other times need only one. I assumed that the added benefit of less options and more "out-of-the-box" outweighs the need to sign-in (which you probably want to do anyway). We also support better auth for non-interactive environments now, so everything should be a lot smoother with this approach.

Reply:
OK